packages/symfony-static-dumper/README.md

```diff
@@ -21,6 +21,12 @@ return [
 ];
 ```
 
+> To make Controller with argument, eg: `/blog/{slug}`, statically dumped, your controler class have to implements `Symplify\SymfonyStaticDumper\Contract\ControllerWithDataProviderInterface` and implements 3 methods:
+> - `getControllerClass()`
+> - `getControllerMethod()`
+> - `getArguments()`
+
+
```

Comment:
This might help, but people would still have to guess the content of the provider.

1) There should be real PHP example too:

- https://github.com/TomasVotruba/tomasvotruba.com/blob/master/packages/symfony-static-dump/src/ControllerWithDataProvider/PostControllerWithDataProvider.php

2) and simple PostController:

- https://github.com/TomasVotruba/tomasvotruba.com/blob/e73eec9fbc314e9efd4e8156ed06e997f626c7dc/packages/blog/src/Controller/PostController.php#L25

Reply:
updated.